nengo_spa/modules/__init__.py

```diff
@@ -0,0 +1,28 @@
+"""SPA modules.
+
+SPA modules derive from `nengo_spa.Network` and thus provide information about
+inputs and outputs that can be used in action rules and the associated
+vocabularies. Note that a module might have no inputs and outputs that can be
```

Comment:
What's an example of a module with no inputs and outputs? It might be worth mentioning here, because it confused the heck out of me.